dataflow/manual/content.tex

```diff
@@ -1141,6 +1141,12 @@ \subsubsection{Default case and fall through for switch statement}
 graph in \autoref{fig:CFGSwitch}. The fake variable named \code{switch#num0}
 is created and each of case nodes creates the branches.
 
+\subsubsection{Break and continue in try block}
+Break and continue statements for while-loop or labeled statements in try
+block will cause executing finally blocks until the target of the jump.
+Dataflow framework will emulate this sort of execution by duplicating
+finally blocks until the target of jump at break and continue statements.
```

Comment:
These two sentences are not very clear. Let me try to rephrase. Please correct if I got something wrong:

```
Break and continue statements within try blocks will cause execution of the finally block
before continuing at the target of the jump.
The Dataflow Framework models this behavior by duplicating the finally block to the
location before the jump to the target of the break or continue statement.
```

I'm not sure what the `until` meant in both sentences.